libkbfs/folder_branch_ops.go

```diff
@@ -4160,13 +4197,22 @@ const (
 const CtxFBOOpID = "FBOID"
 
 func (fbo *folderBranchOps) ctxWithFBOID(ctx context.Context) context.Context {
-	return ctxWithRandomID(ctx, CtxFBOIDKey, CtxFBOOpID, fbo.log)
+	return ctxWithRandomIDReplayable(ctx, CtxFBOIDKey, CtxFBOOpID, fbo.log)
+}
+
+func (fbo *folderBranchOps) newCtxWithFBID() (context.Context, context.CancelFunc) {
+	ctx := NewContextReplayable(context.Background(), fbo.ctxWithFBOID)
```

Comment:
This will assign a random FBOID on a replay, right?  It should be static across replays I think.